app/models/app.rb

```diff
@@ -0,0 +1,16 @@
+class App < ApplicationRecord
+  has_many :app_owners
+  has_many :owners, through: :app_owners, source: :user
```

Comment:
@case-eee Not sure if this is the best way to do this. This is setting up two different relationships between apps and users depending on whether the role is as an app_owner (admin) or as an authorized user.